app/uk/gov/hmrc/partnershipidentificationfrontend/forms/CaptureSautrForm.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentificationfrontend.forms
+
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.data.validation.Constraint
+import uk.gov.hmrc.partnershipidentificationfrontend.forms.utils.ValidationHelper._
+import uk.gov.hmrc.partnershipidentificationfrontend.forms.utils.ConstraintUtil._
+
+import scala.util.matching.Regex
+
+object CaptureSautrForm {
```

Comment:
SoleTrader has a similar form that's a bit simpler, might be worth copying it over